modules/contentpool_normalization/src/EventSubscriber/ContentpoolNormalizationEventSubscriber.php

```diff
@@ -53,6 +53,31 @@ public function onAlterContentData(ReplicationContentDataAlterEvent $event) {
           $paragraph_data[$delta] = $entity->field_paragraphs->get($delta)->entity->toArray();
         }
         $normalized[$key]['field_data'][] = ['value' => json_encode($paragraph_data)];
+
+        if (!$entity->get('field_channel')->isEmpty()) {
+          /** @var \Drupal\taxonomy\Entity\Term $root_channel */
+          $channel = $entity->field_channel->entity;
+          if ($channel->hasField('field_remote_site') &&
```

Comment:
minor, but better keep ifs on a single line. according to code-style it's allowed to have long lines, only break them if they become very long and hard to see else, e.g. if >120chars.